Waveform generation isn't working

Logging error:
```
2024-12-31T12:41:52.059 vibin [ERROR] Error running audiowaveform tool: Input file: /tmp/vibin_snonjbk1d-7349072289512241772
Frames: 21139776
Sample rate: 44100 Hz
Channels: 2
Format: 0x170002
Sections: 1
Seekable: yes
Generating waveform data...
Samples per pixel: 256
Input channels: 2
Output channels: 1
Done: 100%
Read 21139776 frames
Generated 82578 points
Output file: (stdout)
Done
```

--- a/vibin/managers/waveform_manager.py
+++ b/vibin/managers/waveform_manager.py
@@ -97,10 +97,13 @@ class WaveformManager:
                     capture_output=True,
                 )
 
-                if waveform_data.stderr:
-                    raise VibinError(
-                        f"Error running audiowaveform tool: {waveform_data.stderr.decode('utf-8')}"
-                    )
+                if waveform_data.returncode != 0:
+                    error_msg = f"[code: {waveform_data.returncode}]"
+
+                    if waveform_data.stderr:
+                        error_msg += f" {waveform_data.stderr.decode('utf-8')}"
+
+                    raise VibinError(f"Error running audiowaveform tool: {error_msg}")
 
                 if data_format == "json":
                     try:

